application/src/main/java/org/togetherjava/tjbot/features/Features.java

```diff
@@ -72,9 +72,9 @@ public static Collection<Feature> createFeatures(JDA jda, Database database, Con
         ModerationActionsStore actionsStore = new ModerationActionsStore(database);
         ModAuditLogWriter modAuditLogWriter = new ModAuditLogWriter(config);
         ScamHistoryStore scamHistoryStore = new ScamHistoryStore(database);
-        HelpSystemHelper helpSystemHelper = new HelpSystemHelper(config, database);
-        CodeMessageHandler codeMessageHandler = new CodeMessageHandler();
         ChatGptService chatGptService = new ChatGptService(config);
+        HelpSystemHelper helpSystemHelper = new HelpSystemHelper(config, database, chatGptService);
+        CodeMessageHandler codeMessageHandler = new CodeMessageHandler();
```

Comment:
Why this change? Prefer to not change `Features` unless needed (this file causes a lot of merge conflicts with everyone)

Reply:
My understanding is that this allows for only one ChatGPTService object to be created but be passed around the project. The class is not statically accessible. I could refactor it to be or if there is another workaround?

Reply:
The code is fine, but all u did was changing the order:

![foo](https://i.imgur.com/hgSgZGw.png)

And that seems unecessary.

Reply:
Ah, now I see it. Fair.